index.src.html

```diff
@@ -306,6 +307,10 @@ <h3 id="header">
         </pre>
       </div>
 
+      Note: Clearing cookies should also clear the [=Accept-CH cache=] for
+      <a>origin</a>. This is because the cache is also cleared if the user
+      manually clears cookies.
```

Comment:
Perhaps you could also update CH infra to include this requirement? Around https://wicg.github.io/client-hints-infrastructure/#accept-ch-cache-definition:~:text=As%20such%2C%20a%20user%20agent%20MUST%20evict%20that%20cache%20whenever%20the%20user%20clears%20their%20cookies%20or%20when%20session%20cookies%20expire. looks right.

Reply:
I will, was waiting on this first